README.md

```diff
@@ -42,24 +51,26 @@ See the [releasing documentation](RELEASING.md)
 ## Repository structure
 
 These are the main folders within the repository:
+
 * src: The source code of the agent. Each project within this folder targets a specific library, and there is one core project, which is referenced by all other projects.
```

Comment:
Should we turn these directories into code snippets? It might help them stand out more. `src`, `test`, `docs`, etc.

Reply:
Nice suggestion. Done